package.json

```diff
@@ -4813,7 +4813,7 @@
         "description": "A Razor component element"
       },
       {
-        "id": "razorComponentAttribute",
+        "id": "RazorComponentAttribute",
```

Comment:
This is weird, but somehow the Razor language server defines `RazorComponentAttribute` as uppercase but all other tokens as lowercase: https://github.com/dotnet/razor/blob/main/src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Models/RazorSemanticTokensLegend.cs#L31C13-L31C13

We probably want to fix this in the future, although I'm not sure of the implications of making it lowercase/what other things may break

Reply:
Ah you added this as soon as I hit submit review.  IMHO we should modify the server side.  It would only break people that have added specific theme customization for this token.  But I'm not sure that is super common, and it is easily fixable.

Reply:
+1 to making a server fix here, unless there's some timeline issues with flowing the bits.

Reply:
So changing the casing on the server side breaks VS:
![image](https://github.com/dotnet/vscode-csharp/assets/16968319/6c63192b-94a7-47ee-9ce3-9c19b536be64)

I think this is because we have to make a change to the VS LSP client as well: https://devdiv.visualstudio.com/DevDiv/_git/VSLanguageServerClient?path=/src/product/RemoteLanguage/Impl/Features/SemanticTokens/Tagger/SemanticTokensTaggerBase.cs&version=GBdevelop&line=564&lineEnd=565&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

I'm going to file a follow up issue for this, but since the fix requires multiple repos I think we could still take this as a temporary solution